src/screens/FeedScreen/ContentPoll.js

```diff
@@ -7,7 +7,7 @@ import PollOptions from '../../components/PollOptions';
 import PollOptionsMultipleChoice from '../../components/PollOptionsMultipleChoice';
 import useContentPoll from './hooks/useContentPoll';
 import {COLORS} from '../../utils/theme';
-import {fonts, normalizeFontSize, normalizeFontSizeByWidth} from '../../utils/fonts';
+import {fonts, normalize, normalizeFontSize, normalizeFontSizeByWidth} from '../../utils/fonts';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement on line 10 has been modified to include `normalize` which was not present in the old hunk. However, there is no usage of `normalize` in the new hunks provided. If it's not used elsewhere in the file, this import can be removed.
```diff
- import {fonts, normalize, normalizeFontSize, normalizeFontSizeByWidth} from '../../utils/fonts';
+ import {fonts, normalizeFontSize, normalizeFontSizeByWidth} from '../../utils/fonts';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->